utHelper.py

```diff
@@ -0,0 +1,372 @@
+        """
+        Changes the permissions of a folder, file, or path on the target device.
+
+        Args:
+            permission (str): The new permissions to set (e.g., "755").
+            path (str): The path to the folder, file, or path.
+            device (str, optional): The device on which to change permissions (default: "dut").
+        """
+        self.writeMessageToDeviceSession("chmod " + permission + " " + path + "*", device=device)  # Send the 'chmod' command
+        self.writeMessageToDeviceSession("\n", device=device)  # Send a newline
+
+    def copyFileFromHost(self, sourcePath, destinationPath, device="dut"):
+        """
+        Copies a file from the host machine to the target device using SCP (for SSH connections).
+
+        Args:
+            sourcePath (str): The source path and filename on the host.
+            destinationPath (str): The destination path and filename on the device.
+            device (str, optional): The device to copy the file to (default: "dut").
+
+        Returns:
+            str: The message from the subprocess (SCP output).
+
+        Raises:
+            ValueError: If the session type is not "ssh".
+        """
+        activeDevice = self.devices.getDevice(device)
+
+        if activeDevice.session.type == "ssh":
+            self.log.stepMessage("copyFile(" + sourcePath + ", (" + destinationPath + ")")
+
+            # TODO: Assumption: 'root' user is the default on the target, this should be specified by the configuration
+            destination = "root@{}:{}".format(self.slotInfo.getDeviceAddress(), destinationPath)
+
+            # Construct the SCP command with options to disable strict host key checking and known_hosts file
+            command = ["scp", "-o", "StrictHostKeyChecking=no", "-o", "UserKnownHostsFile=/dev/null",
+                    sourcePath, destination]
+
+            # Execute the SCP command and capture the output
+            result = subprocess.run(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+            message = result.stdout.decode('utf-8').strip()
+
+        elif self.session.type == "serial":
+            # self.writeMessageToDeviceSession("cp " + source + " " + destination)  # Commented out code, potentially for serial copy
+            self.log.error("Can't copy for this session type: serial")
+            raise ValueError("Copying files is not supported for serial connections.")
+
+        return message
+
+
+# Device / Host Command operations
+
+    def writeHostCommand(self, command):
+        """
+        Executes a command on the host machine (Linux only).
+
+        Args:
+            command (str): The command to execute.
+
+        Returns:
+            str: The output/result of the command execution.
+        """
+        self.log.debug("writeHostCommand()")
+        result = self.syscmd(command)  # Execute the command using syscmd (assuming it's defined elsewhere)
+        return result
```

Comment:
This function exists in testController already, I believe, though I'm not sure if it's really a valid thing to be doing. Since we're already running python on the host.